ctk/common/src/main/kotlin/org/codice/compliance/utils/TestCommon.kt

```diff
@@ -88,7 +89,15 @@ class TestCommon {
         private val DEPLOY_CL = getDeployDirClassloader()
         private val spMetadata = Common.parseSpMetadata()
 
-        val idpMetadata = Common.parseIdpMetadata()
+        val idpMetadata by lazy {
+            val metadata = Common.parseIdpMetadata()
+            if (metadata.descriptor.supportedProtocols.none { it.contains("2.0") })
```

Comment:
✏️ Might be a good idea to add a TODO here to move this check into the, once completed, metadata verification.
✏️ Also might be a good idea to add a comment specifying what section this check is coming from

Reply:
It doesn't really verify a specific spec section and we might not do metadata testing so I don't wanna polute our code with TODOs we might not so.